prometheus-to-sd/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 The Kubernetes Authors. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM busybox
+
```

Comment:
It looks like this can be run as a non-root user. If that's the case, please add:
```
#   nobody:nobody
USER 65534:65534
```
(e.g. https://github.com/kubernetes/contrib/blob/master/404-server/Dockerfile#L18)